common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -5927,7 +6001,14 @@ schematic = (function() {
 	Source.prototype.constructor = Source;
 
 	Source.prototype.toString = function() {
-	    return '<'+this.type+'source '+this.properties['params']+' ('+this.x+','+this.y+')>';
+	    return edx.StringUtils.interpolate(
+		'<{type}source {params} ({x},{y})>',
+		{
+		    type: this.type,
+		    params: this.properties['params'],
```

Comment:
Note: Hopefully params doesn't include any tags? :)

Reply:
Should not, should be a string.